src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -41,124 +45,144 @@
 
 _T = TypeVar("_T")
 NodeSpans = list[dict[str, Any]]
+SnubaSpan = TypedDict(
+    "SnubaSpan",
+    {
+        "precise.finish_ts": str,
+        "precise.start_ts": str,
+        "problem": IssueOccurrence,
+        "span_id": str,
+        "transaction.id": str,
+    },
+)
 SnubaTransaction = TypedDict(
     "SnubaTransaction",
     {
         "id": str,
+        "issue.ids": list[int],
         "issue_occurrences": Sequence[IssueOccurrence],
-        "transaction.status": int,
-        "transaction.op": str,
-        "transaction.duration": int,
-        "transaction": str,
-        "timestamp": str,
-        "occurrence_spans": Sequence[dict[str, object]],
-        "precise.start_ts": int,
+        "measurements": dict[str, int],
+        "occurrence_id": list[str],
+        "occurrence_spans": list[SnubaSpan],
+        "occurrence_to_issue_id": dict[str, list[int]],
         "precise.finish_ts": int,
-        "trace.span": str,
+        "precise.start_ts": int,
+        "profile.id": str,
+        "profile.profiler_id": str,
```

Comment:
The type here changed from before from

```python
"profile.profile_id" -> "profile.profiler_id"
"profiler.id" -> "profile.id"
```

Can you double check to make sure which one is correct?

Reply:
New one is correct, these types weren't being used before so they were just wrong without being noticed